tests/MongoDB.Analyzer.Tests.Common/DiagnosticTestCaseAttribute.cs

```diff
@@ -126,9 +126,10 @@ public sealed class BuildersMQLAttribute : DiagnosticRuleTestCaseAttribute
         {
         }
 
-        public BuildersMQLAttribute(string message, string version) :
+        public BuildersMQLAttribute(string message, string version, params int[] codeLines) :
             base(DiagnosticRulesConstants.Builders2MQL,
                 message,
+                locations: codeLines.Any() ? codeLines.Select(l => new Location(l, -1)).ToArray() : null,
```

Comment:
technically `codelines` can be null in this case: `BuildersMQLAttribute("", "", null)`.

Reply:
Good catch, done!